content/strategy-goals/strategy/code-graph/search/product.md

```diff
@@ -12,26 +12,33 @@ Quicklinks:
+Expand the core platform functionality with products and features that address our core [use cases](../../../strategy/use-cases/index.md). Sourcegraph needs to continue to differentiate its core platform beyond code search. Search will always power our platform, but to remain competitive we need to become more than the world's best code search. We must provide compelling solutions for the most pressing use cases of our customers. Our goal is to ensure the base Sourcegraph platform (currently Search, Notebooks, and Code monitors) continues to become more compelling and differentiated and supports future innovation from our team and across Sourcegraph.
+
+- Support customers in onboarding new developers and sharing best practices, via Notebooks
+- Enable more connectivity to Sourcegraph via webhooks and Slack integration
+- Continue to improve our core platform experience by continuing to invest in the existing products and features, performance, and experimental R&D efforts that leverage our core search competency.
 
 #### Details
 
-**Search notebooks** will enable individual developers and teams to easily document codebases and share best practices across an organization by enabling a [Jupyter notebooks](https://jupyter.org/)-style experience. Developers will be able to embed search results, code snippets and more, in a markdown-based notebook that creates more robust documentation. For example, using structural search to persitently capture a function and its contents, even if it's moved in the codebase.
+**Search notebooks** achieve product market fit and enable individual developers and teams to easily document codebases and share best practices across an organization by enhancing Notebooks with sharing and duplication functionality, novel Notebook creation tools, and useful templates. Developers can already embed search results, code snippets and more, in a markdown-based notebook that creates more robust documentation. For example, by using structural search to persistently capture a function and its contents, even if it's moved in the codebase.
 
 **Code monitor integrations** will provide the ability to integrate Code monitors with any service that supports webhooks, and will enable multiple actions to be triggered by a Code monitor. We will also support Slack integration with Code monitors.
 
 #### What we're not working on and why
 
-We currently have no development work in the pipeline for search relevance, ranking, or semantic and natural language searching. These are important areas where we do plan to invest substantial research, but we have no implementation plans for FY22Q4.
+We're not going to try and go feature for feature with GitHub code search. That's a game of diminishing marginal returns and takes our focus off of truly differentiating features powered by our superior search technology. That doesn't mean search is no longer a central focus–far from it–but we'll stay true to our vision for the future of code search.
```

Comment:
This is intended more as a list of features that people might have thought you were working on and why. Instead of this, are there key features that GitHub is going after that we don't think are important? That would be interesting to list here and why. That we're generally not cloning GitHub exactly probably is obvious enough to not need to be explicitly said.